src/frontend/shared-components/src/input/__snapshots__/Input.test.js.snap

```diff
@@ -1,6 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Input Editable white input field matches the snapshot 1`] = `
+exports[`Input Input field when mandatory and no label matches the snapshot 1`] = `
+<div>
+  <input
+    className="editable_white"
+    id="textInputId"
+    onChange={[Function]}
+    placeholder="Enter id"
+    type="text"
+  />
+  <br />
+  <span
+    className="error"
+  />
+</div>
+`;
+
+exports[`Input Input field when mandatory and with label matches the snapshot 1`] = `
```

Comment:
same, so perhaps it was meant.

Reply:
nope, it is fixed now - i fixed the wording